ExcelWeb.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.6.33815.320
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "BulkyWeb", "BulkyWeb\BulkyWeb.csproj", "{D0EB1922-3114-4BE2-8549-5F2C4FEBF00F}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ExcelWeb", "BulkyWeb\ExcelWeb.csproj", "{D0EB1922-3114-4BE2-8549-5F2C4FEBF00F}"
```

Comment:
Had an old workspace where I was trying things out, cleaned it up and now we've got a good name.